### DISCUSSION TITLE
Riso Grids: Exercise 2

### DISCUSSION THREAD

**Discussion by sharlagelfand** [posit-conf-2023/creative-coding#8] [Show and tell]
1. Open the file file `exercises/02-riso-grids/exercise-2-3.Rmd`
2. Take the two layers you sketched and turn them into tribbles with columns `row`, `column`, `shape`, and `color`. Try using two of the colors listed in the file, which are common colors in riso printing.
3. Run `generate_and_plot_two_layers()` with your two layers and save the plots in the correct dimension using `ggsave()`
4. Share your two plots in this GitHub discussion

**Comment by Meghansaha** [comment]
![image](https://github.com/posit-conf-2023/creative-coding/assets/67963743/e6b7fe2c-d848-4a04-a626-f11cd3319948)

![image](https://github.com/posit-conf-2023/creative-coding/assets/67963743/fe9ec5b8-fc14-4373-9608-29fe0eb500ec)

**Comment by maxrogow** [comment]
![riso_layer_1](https://github.com/posit-conf-2023/creative-coding/assets/145362959/134f768a-0907-4be6-b35b-9f661932a48a)
![riso_layer_2](https://github.com/posit-conf-2023/creative-coding/assets/145362959/5f4d8f41-b6e5-4fe7-9823-490151fd161e)

**Comment by thomaspara** [comment]
![stuff](https://github.com/posit-conf-2023/creative-coding/assets/10082327/701e1bb1-d4c4-4a5a-bad9-ff66c2423981)
![stuff2](https://github.com/posit-conf-2023/creative-coding/assets/10082327/9a55f40c-97b7-4ee2-aea2-2470175e938b)

**Comment by sshaughnessy** [comment]
![layer2](https://github.com/posit-conf-2023/creative-coding/assets/36434962/8db55360-ba94-4340-ac81-aaccc83cb2fa)
![layer1](https://github.com/posit-conf-2023/creative-coding/assets/36434962/514bc3b3-45b3-4e72-acc8-00601022ba93)
![combined_layers](https://github.com/posit-conf-2023/creative-coding/assets/36434962/5a4ee419-3d5d-4bbf-994e-d3bb54f17c7a)

**Comment by lwdozal** [comment]
![layer_2](https://github.com/posit-conf-2023/creative-coding/assets/19821171/a9910c48-7707-4377-a2ff-ca77b440493e)
![layer_1](https://github.com/posit-conf-2023/creative-coding/assets/19821171/973730f9-4979-41a4-b5aa-a377375ec3b6)

**Comment by maxrogow** [comment]
![risograph](https://github.com/posit-conf-2023/creative-coding/assets/145362959/56570327-8a8c-437c-9ae1-1565907a599e)

**Comment by MHanken** [comment]
![layer1](https://github.com/posit-conf-2023/creative-coding/assets/49319823/6b24c82b-8159-490c-8a85-0b541b95e7c6)
![layer2](https://github.com/posit-conf-2023/creative-coding/assets/49319823/033344c0-13fb-4839-b528-1478ad59b25f)

**Comment by dougtommet** [comment]
![riso-1](https://github.com/posit-conf-2023/creative-coding/assets/7944225/574b1d2d-bbd6-4b40-a933-08eaaf417cec)
![riso-2](https://github.com/posit-conf-2023/creative-coding/assets/7944225/719ecd09-1aab-4d18-a83b-7196f503ecb9)
![riso-combined](https://github.com/posit-conf-2023/creative-coding/assets/7944225/192e8c69-e7e0-4852-bb2c-e21785b68751)
